packages/now-next/test/fixtures/00-trailing-slash-remove/next.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { trailingSlash: false };
```

Comment:
This test starts with `00-`. Did you mean for both to start with the same number?

Reply:
Yes, it was intentional that these start with `00` so they're sorted to the top and run first.